packages/next/src/server/lib/app-info-log.ts

```diff
@@ -23,10 +23,19 @@ export function logStartInfo({
   experimentalFeatures?: ConfiguredExperimentalFeature[]
   maxExperimentalFeatures?: number
 }) {
+  let bundlerSuffix
+  if (process.env.TURBOPACK) {
+    bundlerSuffix = ' (Turbopack)'
+  } else if (process.env.NEXT_RSPACK) {
+    bundlerSuffix = ' (rspack)'
```

Comment:
` (Rspack)` seems more consistent with ` (Turbopack)`

Reply:
does rspack have a specific preference as a project for how it's cased?

Reply:
https://rspack.dev/misc/branding/guideline#the-name always prefer `Rspack`  except as package name